test/core/GnosisSafe.ModuleManager.spec.ts

```diff
@@ -62,7 +62,7 @@ describe("ModuleManager", async () => {
 
             await expect(
                 await safe.getModulesPaginated(AddressOne, 10)
-            ).to.be.deep.equal([[user2.address], AddressOne])
```

Comment:
This text should not have to be adjusted. I would stick to the defined behavior for this case, meaning at the end of the page `AddressOne` is returned as next (signaling the end).

Reply:
> I would stick to the defined behavior for this case, meaning at the end of the page AddressOne is returned as next (signaling the end).

It seems like this would require adjustments in the module manager contract, because unlike `OwnerManager` we don't have the last module pointing to a sentinel address. Do you think it's worth it?

Reply:
> we don't have the last module pointing to a sentinel address.

That should always be the case

Reply:
Do we want to do something like this in `getModulesPaginated` method then:
```
if (next == address(0)) next = SENTINEL_ADDRESS;
```

Reply:
why?

Reply:
Ah, I was confused how this was possible until I saw that `setupModules` is called during the setup. Will investigate why this is not the case